libfuse/fs.go

```diff
@@ -221,6 +223,21 @@ func (r *Root) Lookup(ctx context.Context, req *fuse.LookupRequest, resp *fuse.L
 		return &SpecialReadFile{r.private.fs.remoteStatus.NewSpecialReadFunc}, nil
 	}
 
+	// Special files for macOS/darwin
```

Comment:
Can you move all the platform-specific code to files like `platform_(something_osx.go`, and replace `volIcon` with a `PlatformRoot` type? I think that'll be cleaner.